### PR TITLE
docs: elaborate in readme

### PR DESCRIPTION
**Pull request by mfw78**
This PR:

1. Moves broader documentation from the test suite into the README.md so browsers of repositories can get a better understanding of the repository's context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,46 @@
...
+#### Verification Process
+
+1. Prove that the process can be undertaken from the context of the `NounsDAOExecutor` (ie. the timelock).
+   This means we use `vm.prank` to impersonate the `NounsDAOExecutor` and execute the swap.
+2. Do a full-stack test that:
+   - Proposes the transaction bundle
+   - Vote on the proposal (to affirm).
+   - Queue the proposal (on successful vote).
+   - Execute the proposal (verify transaction bundle executes correctly).
+   - **To be completed:** Verify via settlement that the discrete orders settle correctly.
```
Comment (fleupold) [inline README.md:43]: Do you already have an idea how we can achieve this? Can we "simualte" large internal balances inside the settlement contract and execute a settlement that is served from internal buffers, or do you have another idea?

**Review by fleupold — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (mfw78) [reply README.md:43]: I already have in `ComposableCoW` a test simulation for TWAP, that iterates through blocks, fakes being a solver then automatically produces an opposing `GPv2Trade` for the settlement. For the opposing trade, just a simple `vm.deal` for the total `minBuyAmount`.

Then we just do sanity checks on the safe's `sellToken` balance, and the receiver's `buyToken` balance.

Comment (mfw78) [reply README.md:43]: This is tracked in #2

**Pull request merged**